pom.xml

```diff
@@ -37,6 +37,7 @@
         <module>kafka</module>
         <module>samples</module>
         <module>tools</module>
+        <module>end2end.test</module>
```

Comment:
do you need a separate module ? you could also consider adding it as part of the kafka module since you're e2e tests depend on kafka being the event stream

Reply:
+1

Reply:
If we ever do a Kinesis wrapper (which I expect we will), I'd prefer to see the Kinesis e2e inside of a kinesis module instead of having a single e2e module that knows about all the different wrappers.